modular_skyrat/modules/verbs/code/subtle.dm

```diff
@@ -83,35 +80,46 @@
 	return FALSE
 
 /datum/emote/living/subtler/run_emote(mob/user, params, type_override = null)
+	if(!can_run_emote(user))
+		to_chat(user, "You can't emote at this time.", type = MESSAGE_TYPE_WARNING)
```

Comment:
I really don't know what's the difference between using the `type` parameter and just using a span like is usually done.

Reply:
Alright.